stacks_queues/stack.py

```diff
@@ -21,18 +22,24 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store.head == None:
+            raise StackEmptyException("The stack is empty")
+
+        return self.store.remove_first()
```

Comment:
Per the specification, this should return `None`
```suggestion
        self.store.remove_first()
```